server/src/test/java/org/opensearch/search/fetch/subphase/InnerHitsPhaseTests.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Licensed to Elasticsearch under one or more contributor
```

Comment:
Assuming this is all new content then you don't need this header

Reply:
Will remove